.github/workflows/macos.yaml

```
@@ -33,7 +33,7 @@ jobs:
     - name: Install non-Haskell dependencies (macOS)
       if: contains(matrix.os, 'mac')
       run: |
-        brew update && brew install rocksdb z3 gnu-tar || true
+        brew update && brew install llvm z3 gnu-tar rocksdb || true
```

Why rocksdb, aren't we building it from scratch?

```suggestion
        brew update && brew install llvm z3 gnu-tar || true
```